WordPress/src/main/java/org/wordpress/android/viewmodel/giphy/GiphyPickerDataSource.kt

```diff
@@ -14,7 +11,6 @@ import com.giphy.sdk.core.network.api.GPHApiClient
  * [searchQuery] is changed by the user.
  */
 class GiphyPickerDataSource(
```

Comment:
I'm wondering if this should be renamed to something like `GifPickerDataSource` so once we implement a solution with Tenor, it would be obvious to put it here. I guess it's not that important though.

Reply:
That's a good idea. We could probably do it when we switch to a different provider. Git has some quirkiness when files and folders are renamed. It always seemed harder for me to track the history of those. In this case, we probably want to make it easier to see the history of the files and be able to integrate the new provider a bit faster.